cvmfs/swissknife_check.cc

```diff
@@ -1065,6 +1087,9 @@ int CommandCheck::Main(const swissknife::ArgumentList &args) {
     return 1;
   }
 
+  const shash::Any hash_zero(shash::kMd5);
```

Comment:
```suggestion
  const shash::Any hash_null();
```
zero --> null for consistency with the rest of the code.